jwst/extract_1d/ifu.py

```diff
@@ -56,8 +56,14 @@ def ifu_extract1d(input_model, refname, source_type, subtract_background):
         slitname = "ANY"
 
     extract_params = ifu_extract_parameters(refname, slitname, source_type)
+    sub_bkg = extract_params['subtract_background']
     if subtract_background is not None:
         extract_params['subtract_background'] = subtract_background
+    if subtract_background and not sub_bkg:
```

Comment:
What if the command-line param subtract_background wasn't set at all, i.e. it's None? Will this test fail? Or is it the case that if the user doesn't set it, it still gets assigned a default value (False, I believe) by the step class, so that it's never None in this part of the code? In which case the test in line 60 isn't really necessary (if it can never be None).

Reply:
This should have been indented. Will fix it shortly.